style.css

```diff
@@ -432,3 +441,479 @@ textarea {
+  }
+
+  .flex2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10%;
+  }
+
+  .p-container2 {
+    width: 25%;
+    margin-top: 8%;
+    background-image: url(/assert/Rectangle214.png);
+    background-position: 10% -2%;
+    background-repeat: no-repeat;
+    padding-top: 2%;
+  }
+
+  .projects-info2 {
+    background-image: url(/assert/Rectangle215.png);
+    background-repeat: no-repeat;
+    padding-bottom: 60%;
+    background-size: contain;
+    background-position: bottom right;
+  }
+
+  .article2 {
+    padding: 20px;
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background: #1a2236;
+    padding-top: 4.5%;
+  }
+
+  .project-info-name2 {
+    font-weight: 700;
+    font-size: 60px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    line-height: 80%;
+    padding: 20px 36px 16px 20px;
+    color: #fff;
+  }
+
+  .project-languages2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+    padding: 20px 36px 16px 20px;
+    color: #fff;
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    margin-bottom: 20px;
+    list-style: none;
+  }
+
+  .project-languages2 li {
+    background: #98dede;
+    padding: 6px 10px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 10.3334px;
+    line-height: 80%;
+  }
+
+  .project-container {
+    display: none;
+  }
+
+  .form-section {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .grid2 {
+    display: grid;
+    grid-template-columns: 300px;
+    align-items: center;
+    padding-left: 15%;
+  }
+
+  .container-list1 {
+    flex-direction: column;
+    justify-content: end;
+    padding-left: 100px;
+  }
+
+  #about-container {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-left: 160px;
+    gap: 1rem;
+    margin-bottom: 158px;
+  }
+
+  .sub-about {
+    margin-left: 96px;
+  }
+
+  .about-heading {
+    font-weight: 400;
+    font-size: 110px;
+    color: #1a2236;
+  }
+
+  .section-about {
+    display: grid;
+    grid-template: auto auto/auto auto;
+    background-image: url(assert/image_dots_with_teal.svg);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .section-about .languages {
+    padding: 0;
+    margin: 0;
+    position: relative;
+    width: 289px;
+    height: 289px;
+    font-size: 18px;
+    line-height: 27px;
+  }
+
+  .list1 {
+    padding-top: 51px;
+    padding-left: 50px;
+  }
+
+  .container-list {
+    padding-top: 1px;
+    padding-left: 50px;
+  }
+
+  input {
+    display: flex;
+    width: 100%;
+    height: 52px;
+    background: #fff;
+    border: 1px solid #dfdfe0;
+    box-sizing: border-box;
+    border-radius: 4px;
+    font-size: 0;
+    font-family: "Poppins", sans-serif;
+  }
+
+  #form-section {
```

Comment:
- [ ]  IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element in the page